Zend/zend.c

```diff
@@ -103,6 +136,7 @@ static ZEND_INI_MH(OnUpdateScriptEncoding) /* {{{ */
 ZEND_INI_BEGIN()
 	ZEND_INI_ENTRY("error_reporting",				NULL,		ZEND_INI_ALL,		OnUpdateErrorReporting)
 	STD_ZEND_INI_BOOLEAN("zend.enable_gc",				"1",	ZEND_INI_ALL,		OnUpdateGCEnabled,      gc_enabled,     zend_gc_globals,        gc_globals)
+	ZEND_INI_ENTRY("zend.script_extensions",		".php .phar",		ZEND_INI_ALL,		OnUpdateScriptExtensions)
```

Comment:
If someone put ".php .phar .inc" then, "script.inc" is PHP script. However, move_uploaded_file() will be updated not to allow ".php .phar .inc", so users are protected from script inclusion as long as they use consistent setting.
I forgot about move_uploaded_file() part. I'll add it soon.